### ISSUE TITLE
Page number with total

### ISSUE THREAD

**Issue by yutayamamoto** [yhatt/marp#263]
Hi, it would be great if it has a feature to display page number with total page number.

I'm now doing it manually, (when it has say, 40 pages)

```
<!-- page_number: true -->

<style>
.slide_page::after {
  content: "/40";  
}
</style>
```
Thanks!

**Comment by yhatt** [comment]
We don't intend to update current Marp for this, but it might be valuable for [Marp Next](https://marp.app/).

Currently, our new [Marpit framework](https://marpit.marp.app) supports [styling page number](https://marpit.marp.app/theme-css?id=pagination), but disallows customize `content` even if using `!important`. It will be commented out automatically to avoid wrong use case. However, I think this suggestion might become a mind changer.

An idea for using the number of slides is using data attribute or CSS variables.

```css
/* [NOT WORKING] Planned CSS for Marpit framework */
section::after {
  /* Default */
  /* content: attr(data-marpit-pagination); */

  /* Data attribute (require injecting within Marpit conversion) */
  content: attr(data-marpit-pagination) "/" attr(data-marpit-total);

  /* CSS variables (require injecting within Marpit conversion) */
  content: attr(data-marpit-pagination) "/" var(--marpit-total);
}
```

**Comment by yhatt** [comment]
A just shipped out [Marpit v1.3.0](https://github.com/marp-team/marpit/releases/v1.3.0), the base framework of [Marp Next](https://github.com/marp-team/marp) tools, can use the total page number.
https://marpit.marp.app/theme-css?id=customize-content
